public/sass/components/_timepicker.scss

```diff
@@ -20,6 +20,10 @@
   background-color: $page-bg;
   border-radius: 0 0 0 4px;
   box-shadow: $search-shadow;
+  @include media-breakpoint-down(sm) {
```

Comment:
can you reverse this rule? usually you have responsive first, where the media query is applied for larger screens.